index.html

```diff
@@ -353,6 +353,60 @@
   <section id="conformance">
     <p>There is only one class of product that can claim conformance to this specification: a <dfn>user agent</dfn>.</p>
   </section>
+  
+  <section>
+    <h2>IANA Considerations</h2>
+    <p>The link relation types below have been registered by IANA per Section 6.2.1 of [[!RFC5988]]:</p>
+    
+    <section>
+      <h3>dns-prefetch Link Relation Type</h3>
+      <dl>
+        <dt>Relation Name:</dt>
+        <dd>dns-prefetch</dd>
+        <dt>Description:</dt>
+        <dd>The dns-prefetch link relation type is used to indicate an origin that will be used to fetch required resources, and that the user agent SHOULD resolve as early as possible.</dd>
```

Comment:
On 2016-01-23 13:50, Ilya Grigorik wrote:
> In index.html
> <https://github.com/w3c/resource-hints/pull/53#discussion_r50626188>:
> Feels like we're duplicating the definitions here. Instead, can we link
> to the appropriate sections of the spec? Otherwise, these are guaranteed
> to get out of sync, etc.

you're kind of right. but: the text in the IANA section is the 
description that gets added to the registry, so it's meant to mirror the 
(short) description of the spec. does respec have text macros? that 
might be the best way to avoid things getting out of sync.